eng/pipelines/official.yml

```diff
@@ -1,17 +1,16 @@
 # Pipeline: https://dev.azure.com/dnceng/internal/_build?definitionId=1298
 
-trigger:
-  batch: true
-  branches:
-    include:
-    - main
-    - release/*
-
+# Note: This pipeline can never run automatically because the sourceBranch parameter is required.
+trigger: none
 pr: none
 
 parameters:
-- name: stabilizePackageVersion
-  displayName: Stabilize package version
+- name: sourceBranch
+  displayName: 🚩 Source Branch 🚩
+  type: string
```

Comment:
Since this has no `default`, it is a required parameter. It does correctly validate that the branch exists prior to running the pipeline. This check happens automatically because this parameter is used as part of the repository branch reference.